src/main/java/com/unascribed/fabrication/features/FeatureLegacyCommandSyntax.java

```diff
@@ -58,6 +60,29 @@ public void apply() {
 							.executes(c -> DifficultyCommand.execute(c.getSource(), difficulty)));
 				}
 				dispatcher.register(diffCmd);
+
+				LiteralArgumentBuilder<ServerCommandSource> xpCmd = CommandManager.literal("xp")
```

Comment:
```suggestion
				LiteralArgumentBuilder<ServerCommandSource> xpCmd = CommandManager.literal("experience")
```
I've tested this and it makes this work.

The *L parsing needs work; it's hard to do stuff like this without breaking vanilla client compat by adding a custom argument type.